### ISSUE TITLE
Possible error on documentation of OilRig Infrastructure.md

### ISSUE THREAD

**Issue by blinkomaniak** [center-for-threat-informed-defense/adversary_emulation_library#167]
Hello fellows

Here at [Target Infrastructure](https://github.com/attackevals/ael/blob/main/ManagedServices/oilrig/Emulation_Plan/Infrastructure.md#target-infrastructure) of the OilRig adversary, it says:

`SQL Server : tested and executed on CentOS 7.9`

Which is weird, because down below on the same page, there is the [Configure SQL Server endofroad\ 10.1.0.7](https://github.com/attackevals/ael/blob/main/ManagedServices/oilrig/Emulation_Plan/Infrastructure.md#configure-sql-server-endofroad-10107), all the instructions are for a windows-based machine.

Fortunately, the figure at the bottom specifies that the OS for the SQL Server is Windows Server 2019 Datacenter.

**Comment by kumisaki** [comment]
I got the same problem.

**Comment by blinkomaniak** [comment]
Yeah, it is definitely windows server 2019 datacenter. But At the end I realized that this server is only used for SQL database connection with the Exchange server and what matters is to maintain the connection through 1433 port. It is independent of the OS. But, of course, when you are replicating the infrastructure from zero, these doubts mess with our heads.

**Comment by mgonzcast** [comment]
Yes, that´s one of the things that I came across with the Infrastructure documentation. Others are:

- Some setup files and wrongly linked, like the modify defender powershell script.
- The users and groups are vaguely described how to create them and which permissions to assign.
- The SQL Server mentioned in this thread needs more acuracy in the description. Step 2 is not quite clear to me "Create an "SQL Admins" domain group with tous as a member, additionally giving tous access permissions and ownership of the DB.". Ownership to which database? the master one? Also it needs to state that you need to create a table minfac to import the csv, which by the way, I haven´t managed to import using the SQL management studio.

 I am preparing some scripts to help configuring the setup, can I share them?
